### PR TITLE
Fixed project type passed to WS Master

### LINKED ISSUE #113: Type of created project is wrong

### PR DESCRIPTION
**Pull request by mlabuda**
fixes #113

### CODE REVIEW

File: src/main/java/io/fabric8/che/starter/client/ProjectClient.java [hunk 1]
```diff
@@ -104,6 +104,7 @@ private String getWsAgentUrl (final Workspace workspace) {
     }
 
     private Project initProject(String name, String repo, String branch, String projectType) {
+    	LOG.info("PROOOOOJECT Project type is {}", projectType);
```
Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/client/ProjectClient.java]: PROOOOOOOJECT ???

**Comment by ibuziuk** [comment]
[test]

File: src/main/java/io/fabric8/che/starter/client/ProjectClient.java (hunk 1 above)
Comment (mlabuda) [inline src/main/java/io/fabric8/che/starter/client/ProjectClient.java]: sorry, it was for testing purpose :D

Comment (ibuziuk) [inline src/main/java/io/fabric8/che/starter/client/ProjectClient.java]: @mlabuda I guess you just forgot to remove it ;-)

Comment (mlabuda) [inline src/main/java/io/fabric8/che/starter/client/ProjectClient.java]: yeah, and I made it that way to easy spot it in the log :) fixed

**Pull request merged**